app/helpers/application_helper.rb

```diff
@@ -55,9 +55,11 @@ def conditional_text(variable, prefix = '', suffix = '')
     end
   end
 
+  # This was returing the name, but now returns the estate name -
+  # the top-level organising unit for a particular prison
   def prison_name_for_id(nomis_id)
```

Comment:
@dan-palmer normally I would have, but it's only used in metrics which we're going to refactor soon.  Wanted to keep the effort on this to a minimum.